### PR TITLE
Us34

### PR DESCRIPTION
**Pull request by Unikorn44**
Avatar creation enabled

### CODE REVIEW

**Comment by Unikorn44** [comment]
Lines corrected

**Comment by Unikorn44** [comment]
Code corrected, libe 24 & 25

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
FILE: src/Controller/AvatarsController.php
----------------------------------------------------------------------
FOUND 5 ERRORS AFFECTING 5 LINES
----------------------------------------------------------------------
 24 | ERROR | [x] Opening parenthesis of a multi-line function call
    |       |     must be the last content on the line
 25 | ERROR | [x] Multi-line function call not indented correctly;
    |       |     expected 12 spaces but found 8
 26 | ERROR | [x] Multi-line function call not indented correctly;
    |       |     expected 12 spaces but found 8
 27 | ERROR | [x] Closing parenthesis of a multi-line function call
    |       |     must be on a line by itself
 34 | ERROR | [x] Whitespace found at end of line
----------------------------------------------------------------------
PHPCBF CAN FIX THE 5 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------

**Review by Unikorn44 — COMMENTED** [review]
Chenges approved, added new random function

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
No ! There still are quality issues : 

/home/runner/work/Catch_up/Catch_up/src/Controller/AvatarController.php:13	The variable $stringAléatoire is not named in camelCase.
/home/runner/work/Catch_up/Catch_up/src/Controller/AvatarController.php:13	The variable $stringAléatoire is not named in camelCase.
/home/runner/work/Catch_up/Catch_up/src/Controller/AvatarController.php:32	Avoid variables with short names like $bg. Configured minimum length is 3.


$stringAléatoire --> not accent allowed in a var name

File: src/Controller/AvatarController.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\ConnectManager;
+
+/*explique présence ConnectMAnager*/
+/*session_start();*/
```
Comment (aldaitzwild) [inline src/Controller/AvatarController.php:8]: No dead code !

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
Conflict with dev : 

Conflicting files
src/View/Profiles/profile.html.twig

**Comment by Unikorn44** [comment]
Code modified as requested

**Comment by Unikorn44** [comment]
I HATE AVAST !
Conflicts resolved

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**